### PR TITLE
plugins: configuration-as-code and pipeline-utility-steps

### PR DESCRIPTION
**Pull request by jmarrero**
Update configuration-as-code to 1.54 as it prevented the
Credentials Plugin as others from loading.

Adding pipeline-utility-steps 2.9.0 as is required for the current
Jenkins version to run the pipelines.

Add the coreos-ci-lib global library to the configmap.

### CODE REVIEW

**Review by dustymabe — APPROVED** [review]
trivial LGTM - will let jlebon have the final looksie

File: jenkins/config/coreos-ci-lib.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+unclassified:
+  globalLibraries:
+    libraries:
+      - name: "github.com/coreos/coreos-ci-lib"
```
Comment (jlebon) [inline jenkins/config/coreos-ci-lib.yaml:4]: Shouldn't this just be e.g. `coreos-ci-lib`?

**Review by jlebon — APPROVED** [review]
One comment, but LGTM overall. Thanks!

File: jenkins/config/coreos-ci-lib.yaml (hunk 1 above)
Comment (jmarrero) [reply jenkins/config/coreos-ci-lib.yaml:4]: The error I got said: `ERROR: Could not find any definition of libraries [github.com/coreos/coreos-ci-lib@main]` changing this to `github.com/coreos/coreos-ci-lib` made the pipeline move. I can try with `coreos-ci-lib` and report back what happens.

Comment (jmarrero) [reply jenkins/config/coreos-ci-lib.yaml]: tested with coreos-ci-lib and it also works. Updated

**Review by jlebon — APPROVED** [review]

**Comment by jlebon** [comment]
@jmarrero Mind recreating the configmap and killing the Jenkins pod in the prod environment? Otherwise I think jobs referencing coreos-ci-lib helpers will start failing.

**Comment by jmarrero** [comment]
@jlebon sure thing

**Pull request merged**